system/logrotate.d/haproxy.conf

```diff
@@ -0,0 +1,12 @@
+/var/log/haproxy.log {
```

Comment:
@x974 shouldn't this be /var/log/haproxy/haproxy.log to be in sync with 20-haproxy.conf ?
we should also make sure directory gets created.